src/arr/compiler/type-check-structs.arr

```diff
@@ -756,7 +754,7 @@ fun solve-helper-refinements(system :: ConstraintSystem, solution :: ConstraintS
             end, {system; empty-tree-set})
 
             solve-helper-constraints(temp-system, constraint-solution(empty-tree-set, [string-dict: ]), context).bind(lam({shadow temp-system; temp-solution}, shadow context):
-              cases(ConstraintSolution) temp-solution:
+              cases(ConstraintSolution) temp-solution block:
```

Comment:
Why the `block`?

Reply:
I just missed removing that from my debugging.